src/pkg/packager/prepare.go

```diff
@@ -37,6 +38,7 @@ type imageMap map[string]bool
 func (p *Packager) FindImages() (imgMap map[string][]string, err error) {
```

Comment:
Will need to add `whyResources := []string{}` as well

Reply:
done !